packages/sources/svelte/src/components/structure/VtmnListItem/VtmnListItem.svelte

```diff
@@ -89,12 +101,18 @@
       tabindex={disabled && -1}
       {href}
       {target}
+      on:click|stopPropagation
+      on:keydown={(e) => e.stopPropagation()}
       rel={computeRel(target, rel)}
       aria-label={$$restProps['aria-label']}
       aria-disabled={disabled}
     >
       {#if checkSlotExists('start-visual')}
-        <div class="vtmn-list_start-visual">
+        <div
+          class="vtmn-list_start-visual"
+          on:click={(e) => e.stopPropagation()}
```

Comment:
Do you try to a `pointer-event:none` instead stop propagation ?

Reply:
No, we wants to stop the propagation of parent click event here by default, and just trigger the click on the child.
`point-event: none` will disable every children action.

Reply:
maybe we can discuss that because i'm not sure to understand well

Reply:
This issue are resolved, we don't apply propagation on the `VtmnListItem`.
Because of the behaviour of the component, we have to set manually the `stopPropagation` on the slot's element.